tests/zfs-tests/tests/functional/cli_root/zfs_destroy/zfs_destroy_common.kshlib

```diff
@@ -97,7 +97,7 @@ function setup_testenv #[dtst]
 #
 function cleanup_testenv
 {
-	if is_global_zone && ismounted "$TESTDIR1" "ufs" ; then
+	if is_global_zone && ! ismounted "$TESTDIR1" "zfs" ; then
```

Comment:
For portability `NEWFS_DEFAULT_FS` in place of "ufs".  It's defined to be "ext2" for Linux and is commonly used when a non-zfs filesystem needs to be created on a volume.